.gitignore

```
@@ -1,2 +1,6 @@
 .php_cs.cache
 vendor
+config.json
+plugin.json
+ucrm.json
+plugin.log
```

```suggestion
ucrm.json
```
only `ucrm.json` should be ignored, the rest can be actually included